index.bs

```diff
@@ -904,11 +904,19 @@ and controlling remote playback of media as defined by the
 APIs in [[REMOTE-PLAYBACK|Remote Playback API]] map to the protocol messages
 defined in this section.
 
-If an [=Open Screen Protocol agent=] is a [=controller=] for the Remote Playback
-API, it must advertise the `control-remote-playback` capability.  If an OSP
-agent is a [=receiver=] for the Remote Playback API, it must advertise the
-`receive-remote-playback` capability.  The same agent may be a remote playback
-controller and receiver.
+An [=Open Screen Protocol agent=] that implements the
+[[REMOTE-PLAYBACK|Remote Playback API]] must support the
+`control-remote-playback` capability.  It may support the `send-streaming`
+capability so it can send {{HTMLMediaElement}} media data through media
+remoting.
```

Comment:
This sentence doesn't make much sense until we land the remoting PR.  But I'm fine landing it now since the remoting PR should land soon (right?).

Reply:
We've already updated the charter and the requirements to include remoting.  So I think it's okay to add this now.